### PR TITLE
docs : updated readme.md

### PR DESCRIPTION
**Pull request by hemantjadon**
updated readme.md for documentation of styling  `--iron-icon-width` and `--iron-icon-height`. 

In reference to #87.

### CODE REVIEW

**Comment by hemantjadon** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -72,6 +72,8 @@ The following custom properties and mixins are available for styling:
 | --- | --- | --- |
 | `--paper-icon-button-disabled-text` | The color of the disabled button | `--disabled-text-color` |
 | `--paper-icon-button-ink-color` | Selected/focus ripple color | `--primary-text-color` |
+| `--iron-icon-width` | Width of iron-icon | `100%` |
```
Comment (notwaldorf) [inline README.md]: These are not needed -- they're properties that are exposed by `iron-icon`: https://github.com/PolymerElements/iron-icon/blob/master/iron-icon.html#L72

I would actually just add a line somewhere before this section, along the lines of: "To style the internal icon, please see the`Polymer.IronIcon` documentation"

**Comment by hemantjadon** [comment]
Yeah you are right.
These properties are exposed by `iron-icon` but it is not mentioned anywhere in the docs of `paper-icon-button` how to style these.

**Comment by notwaldorf** [comment]
Thanks, this is perfect!

**Pull request merged**